### ISSUE TITLE
Issue tracker menu item alignment might be off

### ISSUE THREAD

**Issue by zajrik** [zajrik/zajrikSRV-mobile#2]
I haven't done the pixel measurements yet but I swear the menu item for the issue tracker is too low. I'll measure it as soon as I can. Opening this issue as a reminder to myself to look into it.

**Comment by zajrik** [comment]
Did the measurements. The distance between normal items is 58 pixels. The distance between Changelogs and Issue Tracking is 62 pixels. This will be adjusted in the next release.

**Comment by zajrik** [comment]
Fixed as of commit 6226c5b07449af6160bb5c96370da9e925106474. Will be published in v1.0.1

**Comment by zajrik** [comment]
I realized that as this change was made in pixels, I am uncertain of how it will look on devices of different DPIs. I only have access to one device (or rather 2 of the same device, counting my wife's handset) of an appropriate Android version for testing purposes, and I've found that emulating devices tends to have different visual results versus physical devices. Until I can test this on a device of a different DPI, I'll have to assume the adjustment I've made will scale differently, and thus displace the menu item for that device.

Or, I can just put on the rose-colored glasses and assume everyone has an xhdpi handset and I'll be worry-free.
